mezzanine/project_template/deploy/local_settings.py.template

```diff
@@ -15,7 +15,7 @@ DATABASES = {
         # Not used with sqlite3.
         "PASSWORD": "%(db_pass)s",
         # Set to empty string for localhost. Not used with sqlite3.
-        "HOST": "127.0.0.1",
+        "HOST": "localhost",
```

Comment:
I don't remember exactly why I went with `localhost` (it was related to some DB connection issues), but I'll test it and change it back to `127.0.0.1` if I can't reproduce the problem.